sys-process/procps/procps-4.0.0-r1.ebuild

```diff
@@ -0,0 +1,100 @@
+# Copyright 2022 Lucas Roberto <sigmw@protonmail.com>
```

Comment:
Comments:
- Please don't change the copyright header.
- Please sign off your commits (Signed-off-by: real name <email>)
- Use 'cat/pkg:' prefix in commit message (so here, it'd be: 'sys-process/procps: add 4.0.0', given the musl thing is sort of implied with the bump)